assets/dev/js/editor/components/documents/models/editor.js

```diff
@@ -0,0 +1,43 @@
+export default class Editor {
```

Comment:
File name should be `models/editor.js` or `modules/editor.js`?

Reply:
its model.